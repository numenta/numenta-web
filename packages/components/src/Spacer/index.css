/**
 * Numenta Web Platform and Sites source code
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2017 Numenta <http://numenta.com>
 */

.spacer {
  composes: mh1 from 'tachyons';
  composes: silver from 'tachyons';
}
