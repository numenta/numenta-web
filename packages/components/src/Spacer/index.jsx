// Numenta Web Platform and Sites source code
// MIT License (see LICENSE.txt)
// Copyright © 2005—2017 Numenta <http://numenta.com>

import React from 'react'

import styles from './index.css'


const Spacer = () => (
  <span className={styles.spacer}>•</span>
)

export default Spacer
