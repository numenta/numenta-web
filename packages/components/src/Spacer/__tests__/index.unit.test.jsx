import React from 'react'
import renderer from 'react-test-renderer'

import Spacer from '../../Spacer'


describe('Spacer React component', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <Spacer />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
