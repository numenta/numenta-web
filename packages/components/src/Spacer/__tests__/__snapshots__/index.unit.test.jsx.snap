exports[`Spacer React component Renders correctly 1`] = `
<span
  className="spacer">
  â€¢
</span>
`;
